pkg/elasticsearch/client.go

```diff
@@ -47,10 +48,19 @@ type Client interface {
 	GetClusterNodeCount() (int32, error)
 
 	// Index API
+	GetIndex(name string) (*estypes.Index, error)
 	CreateIndex(name string, index *estypes.Index) error
+	ReIndex(src, dst, script, lang string) error
+
+	// Inde Alias API
 	ListIndicesForAlias(aliasPattern string) ([]string, error)
+	Aliases(actions estypes.AliasActions) error
```

Comment:
without having looked at the implementation yet, I'm not sure what this function does based on the name... is this a getter?

Reply:
I kept the method naming as `aliases` because it represents the [index aliases API](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/indices-aliases.html) `_aliases` one to one.

Reply:
right, but every other function of ours is descriptive in what it is doing like "List Indices for Alias" or "Add Alias for Old Indices"

Reply:
Renamed to `UpdateAlias`